broadcaster/message/message.go

```diff
@@ -52,8 +57,33 @@ func (m *BroadcastFeedMessage) UpdateCumulativeSumMsgSize(val uint64) {
 	m.CumulativeSumMsgSize += val + m.Size()
 }
 
-func (m *BroadcastFeedMessage) Hash(chainId uint64) (common.Hash, error) {
-	return m.Message.Hash(m.SequenceNumber, chainId)
+// SignaturePreimage creates a preimage for the feed signature that include all fields that need to
+// be signed. Be aware that changing this function can break compatibility with older clients.
+func (m *BroadcastFeedMessage) SignaturePreimage(chainId uint64) common.Hash {
```

Comment:
We usually use the term "Preimage" to meansomething like  "a hash that can be used to fetch the data that it hashes".

Maybe name something like "SignatureHash" or "IdentifyingHash" or "PartialHash" or something?